test/Driver/EDMF/closures/mixing_length.jl

```diff
@@ -0,0 +1,119 @@
+#### Mixing length model kernels
+
+function mixing_length(
+    ss::SingleStack{FT, N},
+    m::MixingLengthModel,
+    source::Vars,
+    state::Vars,
+    diffusive::Vars,
+    aux::Vars,
+    t::Real,
+    direction,
+    δ::FT,
+    εt::FT,
+) where {FT, N}
+
+
+    # Q - do I need to define L as a vector ?
+    # need to code the functions: obukhov_length, ustar, ϕ_m, lamb_smooth_minimum
+
+    # Alias convention:
+    gm = state
+    en = state
+    up = state.edmf.updraft
+    gm_s = source
+    en_s = source
+    up_s = source.edmf.updraft
+    en_d = state.edmf.environment.diffusive
+
+    z = gm_a.z
+    # Parameters
+    cv_d::FT = cv_d(param_set)
```

Comment:
```suggestion
    _cv_d::FT = cv_d(param_set)
```

Reply:
We should use some convention to create these aliases. I suggest a leading underscore, and using the same name of the function, for now. This happens in a few places, so please adjust everywhere.